docs/using-the-jdbc-driver/using-plugins/UsingTheSimpleReadWriteSplittingPlugin.md

```diff
@@ -0,0 +1,78 @@
+properties.setProperty("srwReadEndpoint", "test-db.cluster-ro-XYZ.us-east-2.rds.amazonaws.com");
+```
+
+If you would like to use the simple read/write splitting plugin without the failover plugin, make sure you have the `srw` plugin in the `wrapperPlugins` property, and that the failover plugin is not part of it.
+```
+final Properties properties = new Properties();
+properties.setProperty(PropertyDefinition.PLUGINS.name, "srw");
+properties.setProperty("srwWriteEndpoint", "test-db.cluster-XYZ.us-east-2.rds.amazonaws.com");
+properties.setProperty("srwReadEndpoint", "test-db.cluster-ro-XYZ.us-east-2.rds.amazonaws.com");
+```
+
+## Federated Authentication Plugin Parameters
+| Parameter                 |  Value  | Required | Description                                                                                   | Default Value | Example Value                                                |
+|---------------------------|:-------:|:--------:|:----------------------------------------------------------------------------------------------|---------------|--------------------------------------------------------------|
+| `srwWriteEndpoint`        | String  |   Yes    | The endpoint to connect to when `setReadOnly(false)` is called.                               | `null`        | `<cluster-name>.cluster-<XYZ>.<region>.rds.amazonaws.com`    |
+| `srwReadEndpoint`         | String  |    No    | The endpoint to connect to when `setReadOnly(true)` is called.                                | `null`        | `<cluster-name>.cluster-ro-<XYZ>.<region>.rds.amazonaws.com` |
+| `verifyNewSrwConnections` | Boolean |    No    | Enables role-verification for new connections made by the simple read/write splitting plugin. | `true`        | `true`, `false`                                              |
+
+### Limitations of `verifyNewSrwConnections`
+
+The property `verifyNewSrwConnections` is enabled by default. This means that when new connections are made with the simple read/write splitting plugin, a query is sent to the new connection to verify its role. If a write connection connects to a reader, an error is thrown. If a read connection connects to the writer, a warning is logged and the connection is not stored for reuse.
+
+#### Non-Aurora clusters
+The verification step uses an Aurora specific query to determine the role of the connection, if the endpoint is not to an Aurora database, disable `verifyNewSrwConnections` by setting it to `false` in the properties.
+
+#### Autocommit
+If `verifyNewSrwConnections` is enabled and autocommit is set to false when the simple read/write splitting plugin makes a new connection, unexpected errors such as `Cannot change transaction read-only property in the middle of a transaction.` will occur. As part of verification, the plugin executes a query against a new connection, which when autocommit is set to false will open a transaction. 
```

Comment:
Can we rephrase it? It's hard to understand when the occurs and what to do to fix it.